.changeset/unlucky-walls-tie.md

```diff
@@ -1,5 +1,7 @@
 ---
-"@coinbase/onchainkit": minor
+"@coinbase/onchainkit": patch
 ---
 
-**feat**: Swap success state - refetch balances and clear inputs by @0xAlec #1089
+- **feat**: refetch balances and clear inputs after Swap succeeds. By @0xAlec #1089
```

Comment:
This is not a `feat`, this is a `fix`.

Please let's take the time to do this well. Let's give respect to the docs and changelog.

Reply:
changed in latest